tpm_test.go

```diff
@@ -59,7 +59,7 @@ var (
 	testEncodedEkCertChain []byte
 )
 
-func decodeHexString(t *testing.T, s string) []byte {
+func decodeHexStringT(t *testing.T, s string) []byte {
```

Comment:
my suggestion (no blocker) in this sort of situation given that we expect this to succeed in most cases, and is not really code that in itself needs to be tested, is to have a single helper that panics on error, in the spirit basically of must* helpers in the stdlib (not saying it must be called must* in this case)

Reply:
This is temporary - there will eventually be a single helper once the rest of the tests are using gopkg.in/check.v1.